File: src/erl/comparison_operators.erl
-module(comparison_operators).
-export([start/0]).

start() -> 
    put(result, 1 == 2),
    io:fwrite("1 equals 2 is ~w~n", [get(result)]),

    put(result, 1 /= 2),
    io:fwrite("1 not equals 2 is ~w~n", [get(result)]),

    put(result, 1 > 2),
    io:fwrite("1 larger than 2 is ~w~n", [get(result)]),

    put(result, 1 >= 2),
    io:fwrite("1 larger than or equals 2 is ~w~n", [get(result)]),

    put(result, 1 < 2),
    io:fwrite("1 less than 2 is ~w~n", [get(result)]),

    put(result, 1 =< 2),
    io:fwrite("1 less than or equeals 2 is ~w", [get(result)]),

    erlang:halt().

File: src/ml/comparison_operators.ml
let result = ref (1 = 2);;
Printf.printf "1 equals to 2 is %s\n" (if !result then "True" else "False");;

result := (1 <> 2);;
Printf.printf "1 not equals to 2 is %s\n" (if !result then "True" else "False");;

result := (1 > 2);;
Printf.printf "1 larger than 2 is %s\n" (if !result then "True" else "False");;

result := (1 >= 2);;
Printf.printf "1 larger than or equals 2 is %s\n" (if !result then "True" else "False");;

result := (1 < 2);;
Printf.printf "1 less than 2 is %s\n" (if !result then "True" else "False");;

result := (1 <= 2);;
Printf.printf "1 less than or equals 2 is %s\n" (if !result then "True" else "False");;